src/USER-CAC/compute_CAC_quad_count.cpp

```diff
@@ -0,0 +1,235 @@
+/* ----------------------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   http://lammps.sandia.gov, Sandia National Laboratories
+   Steve Plimpton, sjplimp@sandia.gov
+
+   Copyright (2003) Sandia Corporation.  Under the terms of Contract
+   DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+   certain rights in this software.  This software is distributed under
+   the GNU General Public License.
+
+   See the README file in the top-level LAMMPS directory.
+------------------------------------------------------------------------- */
+
+#include <string.h>
+#include "compute_CAC_quad_count.h"
+#include "atom.h"
+#include "update.h"
+#include "modify.h"
+#include "comm.h"
+#include "force.h"
+#include "neighbor.h"
+#include "pair.h"
+#include "memory.h"
+#include "error.h"
+#include <math.h>
```

Comment:
this should be `<cmath>` and a few lines above the include should pull `<cstring>`. same for **all** files. also the system include files should be included first.